react/features/device-selection/components/VideoInputPreview.js

```diff
@@ -114,6 +123,7 @@ class VideoInputPreview extends Component {
             <div
                 className = 'video-input-preview'
                 ref = { this._setRootElement }>
+                <FaceTracker />
```

Comment:
We should make sure that enableFaceTracking func is set. It's not mandatory, so if it doesn't exist do not add the component.